templates/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+All Submissions:
+
+* [ ] Have you followed the guidelines in our Contributing document?
+* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
```

Comment:
Cool! Didn't think this would work, but just tried it by opening the README! CC @wesrobin.

But, I'm wondering if it would just need to be one folder fewer up, if this file goes in at the root of each repo, rather than in a templates folder? 
```suggestion
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
```